### PR TITLE
Upgrade to Rails 4.2.10

### PR DESCRIPTION
**Pull request by davidwinter**
Specs and Ruby linting is now passing and have verified this locally

### CODE REVIEW

**Comment by h-lame** [comment]
Hi @davidwinter the easiest way to run the app against real data and apis is to use the development VM - all the details about that are here https://docs.publishing.service.gov.uk/manual/get-started.html

File: Gemfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.7.1'
+gem 'rails', '~> 4.2.10'
```
Comment (tijmenb) [inline Gemfile]: Can we pin this to the actual version like in other Rails app?

**Comment by davidwinter** [comment]
We’re going to pull these lint changes out of this PR: https://github.com/alphagov/manuals-frontend/pull/242/files#diff-3c763ac78c7a85c70fd1f70e5382b1bf

We were scratching our heads on how the previous builds had passed the Jenkins builds, but seems like Jenkins only runs linting against the git diffs, which means there are probably offending files in a few repos. 

We’ll submit separate PR’s when we stumble across this in future, and pull these changes out of this PR.

@tijmenb ^

**Comment by danielburnley** [comment]
The file with the lint changes was deleted on master in #243 - So this PR now only contains changes related to the rails update

**Pull request merged**